recipes/conversation_view/message_input/message_input.vue

```diff
@@ -166,19 +167,40 @@
             </template>
           </dt-tooltip>
 
+          <!-- Cancel button for edit mode -->
+          <dt-tooltip
+            v-if="isEdit"
+            :message="cancelTooltipMessage"
+            :offset="[0, -8]"
+          >
+            <template #anchor>
+              <dt-button
+                data-qa="dt-message-input-cancel-button"
+                class="dt-message-input--cancel-button"
+                size="sm"
+                importance="clear"
+                :aria-label="cancelButtonAriaLabel"
+                @click="onCancel"
+              >
+                <p>{{ cancelButtonText }}</p>
+              </dt-button>
+            </template>
+          </dt-tooltip>
+
+          <!-- Send button -->
           <dt-tooltip
             placement="top-end"
             :message="sendTooltipLabel"
             :show="!isSendDisabled && sendButtonFocus"
-            :offset="[6, -4]"
+            :offset="[6, -8]"
           >
             <template #anchor>
               <!-- Right positioned UI - send button -->
               <dt-button
                 data-qa="dt-message-input-send-btn"
                 size="sm"
                 :kind="!isSendDisabled ? 'default' : 'muted'"
```

Comment:
Will suggest to create a computed property here with the name of `buttonKind`

Reply:
done